### PR TITLE
Update readme

### LINKED ISSUE #43: Link to the docker image in Prerequisites is wrong
### Minor issue

Link to the docker image in Prerequisites is wrong, it looks like it is missing the `http://` part

```
It is recommended to use the provided Docker image which can be [found on Quay](quay.io/helmpack/chart-testing/).
It comes with all necessary tools installed.
```

### PR DESCRIPTION
**Pull request by nvtkaszpir**
* fix urls so that they are absolute
* add link to docker tags
* minor formatting
Fixes #43

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,7 +85,10 @@ chart-dirs:
 build-id: pr-42
 ```
 
-`ct install --config config.yaml`
+Config usage:
```
Comment (scottrigby) [inline README.md]: For consistency change this to:

```suggestion
#### Config usage:
```

File: README.md [hunk 2]
```diff
@@ -27,7 +27,8 @@ Unpack the `ct` binary, add it to your PATH, and you are good to go!
 
 ### Docker Image
 
-A Docker image is available at `quay.io/helmpack/chart-testing`.
+A Docker image is available at `quay.io/helmpack/chart-testing` with list of
+avaliable tags [here](https://quay.io/repository/helmpack/chart-testing?tab=tags).
```
Comment (unguiculus) [inline README.md]: avaliable -> available

File: README.md (hunk 1 above)
Comment (unguiculus) [reply README.md]: Title case, please: `Config Usage`

File: README.md [hunk 3]
```diff
@@ -84,7 +85,10 @@ chart-dirs:
 build-id: pr-42
 ```
 
-`ct install --config config.yaml`
+Config usage:
+
+    ct install --config config.yaml
+
```
Comment (unguiculus) [inline README.md]: We should probably add that, if no config file is specified, `ct.yaml` (or any of the supported formats) is loaded from the current directory, `$HOME/.ct`, or `/etc/ct`, in that order, if found.

File: README.md (hunk 1 above)
Comment (scottrigby) [reply README.md]: Nice catch - updated suggestion (my mind always looks for `Sentence case` 🤷‍♂️ )

File: README.md [hunk 4]
```diff
@@ -84,10 +85,16 @@ chart-dirs:
 build-id: pr-42
 ```
 
-`ct install --config config.yaml`
+#### Config Usage:
```
Comment (scottrigby) [inline README.md]: Nit, remove the `:`

**Pull request merged**